static/app/views/monitors/utils.tsx

```diff
@@ -32,11 +32,12 @@ export function makeMonitorListQueryKey(
 
 export function makeMonitorDetailsQueryKey(
   organization: Organization,
+  projectId: string,
```

Comment:
what's the current ruling on this? Do we just use projectId for slug for consistency?